src/selectorNode/SelectorNode.js

```diff
@@ -0,0 +1,86 @@
+import { svgns } from '../canvas/Canvas';
+
+export class SelectorNode {
```

Comment:
what's a selectornode..

Reply:
a selector node are those blue nodes that are used to resize the shape

didn't call it node, coz Node.

open to changing this lame name tho :)